src/Menu/MenuItem.js

```diff
@@ -3,6 +3,7 @@ import PropTypes from 'prop-types';
 import { hideVisually } from 'polished';
 
 import { ENTER_KEY, SPACEBAR } from '../constants';
+import { fontSizes } from '../theme';
```

Comment:
I am not sure if this is the right way to share this value, but Tooltip.js also imports from the theme

Reply:
The issue with importing `fontSizes` like this is that if a user were to override the theme, providing their own theme, this would still reference the main arbor theme font sizes as opposed to the font sizes from their own theme.

Tooltip was a special case where there's no way to access the theme when injecting global styles.

I think the best way to achieve what you're looking to do one of the following:

1. Use the `css` prop from emotion which should have access to the theme provided by the emotion ThemeProivder

2. Wrap the component with the `withTheme` HOC to have the theme passed into our component via the `ThemeProvider` so you can access it that way.

Both of these are documented here - https://emotion.sh/docs/emotion-theming

Reply:
As it turns out, it seems that GlobalStyles now has the theme passed to it's style prop which means we no longer need to import it directly - https://github.com/CatchRelease/arbor/pull/88

Reply:
i'm just going to use the css prop in this case, since it's very specific to this component, in a single place.